README.md

```diff
@@ -202,6 +203,44 @@ Where "dir" is a subdirectory within the repo to push all the check files. Setup
 4. Create `updates/.gitkeep`.
 5. Enable GitHub Pages in your settings for your desired branch.
 
+#### SQL Storage (sqlite3/PostgreSQL)
+
+**[godoc: SQL](https://godoc.org/github.com/sourcegraph/checkup#SQL)**
+
+Postgres or sqlite3 databases can be used as storage backends.
+
+sqlite database file configuration:
+```json
+{
+	"provider": "sql",
+	"sqlite_db_file": "/path/to/your/sqlite.db"
+}
+```
+
+sqlite database file configuration:
+```json
+{
+	"provider": "sql",
+	"postgresql": {
+		"user": "postgres",
+		"dbname": "dbname",
+		"host": "localhost",
+		"port": 5432,
+		"password": "password",
+		"sslmode": "disable"
+	}
+}
+```
+
+The SQL engine used depends on which one is configured; if both sqlite and Postgres configurations are present, sqlite is used.
```

Comment:
Should we treat dual SQL configurations in the same checker as an error?

Reply:
That's probably a good idea, yes.

Reply:
@mholt That should be fixed now.